Formula/opa.rb

```diff
@@ -0,0 +1,27 @@
+class Opa < Formula
+  desc "Open source, general-purpose policy engine"
+  homepage "https://www.openpolicyagent.org"
+  url "https://github.com/open-policy-agent/opa/archive/v0.8.2.tar.gz"
+  sha256 "475cecc8cc12bddfe6d58311e3e084fb1583647b1902c8955d45adba1d22a273"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["PATH"] = ENV["PATH"] + ":" + buildpath + "/bin"
+
+    dir = buildpath/"src/github.com/open-policy-agent/opa"
+    dir.install Dir["*"]
+
+    system "make", "deps", "-C", dir
+    system "make", "-C", dir
+
+    opa_bin = dir/"opa_darwin_amd64"
```

Comment:
`bin.install "opa_darwin_amd64" => "opa"`, although arguably here either we should be building it ourselves without the `Makefile` or upstream should be providing a `make install` step.